packages/client/hmi-client/src/components/widgets/tera-infinite-canvas.vue

```diff
@@ -164,14 +171,24 @@ main > * {
 
 .background-layer {
 	cursor: grab;
+	background-color: var(--surface-0);
 }
 
```

Comment:
Once you put the toolbar in the foreground slot add this to make the toolbar stretch properly.
```
.foreground-layer {
	position: relative;
}
```